MainTel/MainTel-Troubleshooting/MTM2-Troubleshooting/MTM2_recovery.rst

```diff
@@ -0,0 +1,306 @@
+
+#. Move CSC to ``ENABLED``:
+
+    #. Once the CSC is in ``Disabled`` wait 2 minutes.
+
+    #. Transition the CSC to ``Enabled`` using enable, this may take up to 2 minutes.
+
+    #. If it fails, move the CSC all the way to ``standby`` and try to transition to ``enabled`` again with the same wait times.
+
+#. Check that M2 in under Close Loop control **4** in Chronograf `M2 status`_.
+
+    #. If needed, set Closed Loop control by following the ``run_command`` configuration found in `LOVE MT Useful lower-level command scripts + configurations`_.
+
+    This can be done even if you are already under Closed Loop control.
+
+
+
+.. CORRECT Links on how to access MtM2 GUI and Power cycle M2 cabinet, they should be in obs-ops, but not available as of June 2024.
+
+.. _`How to access the MTM2 EUI`: https://confluence.lsstcorp.org/display/OOD/How+to+access+the+MTM2+EUI
+
+.. _`power cycle M2 cabinet`: https://confluence.lsstcorp.org/display/OOD/Power+cycle+MTM2+cabinet?src=contextnavpagetreemode
+
+.. _`LOVE MT Useful lower-level command scripts + configurations`: https://confluence.lsstcorp.org/pages/viewpage.action?pageId=239409017
+
+
+.. _MTM2-recovery-Condition-A-for-Step-4: 
+
+
+Post-Condition
+^^^^^^^^^^^^^^
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- MTM2 is in ``ENABLED`` state. 
+- MTM2 is in Close Loop state (4).
+
+
+.. figure:: _static/MTM2_recovery_state_chronograf.png
+    :name: MTM2 state in chronograf
+    :width: 600
+
+.. note::
+
+    There will be an indicator added in the MTM2 `LOVE view`_ (see that it is missing in the image below), check `LOVE-300`_.
+
+    .. figure:: _static/MTM2_recovery_LOVE_M2.png
+        :name: MTM2 display in LOVE
+        :width: 600
+
+.. _`LOVE-300`: https://rubinobs.atlassian.net/jira/software/c/projects/LOVE/issues/LOVE-300
+
+.. _MTM2-recovery-Contingency:
+
+Contingency
+^^^^^^^^^^^
+
+If you are unable to find the fault, check the cRIO controller log that contains detailed report faults. These logs are found in the `/u/log/` directory. You can list logs with a command like `ls -lrt` which will list the logs with the most recent modified on the bottom. Logs are named by the time and date they were created. You can grab error messages from the log with a command like grep -nr "error" nameofloghere
```

Comment:
Please use one sentence per line. In-line commands like ``ls -lrt`` can be represented in reST with https://obs-ops.lsst.io/project/observatory-operations-reST-reference.html#shell-commands